test/sharness/t0053-dag.sh

```diff
@@ -271,10 +316,10 @@ test_dag_cmd() {
 
   test_expect_success "dag stat of simple IPLD object" '
     ipfs dag stat $NESTED_HASH > actual_stat_inner_ipld_obj &&
-    echo "Size: 15, NumBlocks: 1" > exp_stat_inner_ipld_obj &&
+    echo "Size: 8, NumBlocks: 1" > exp_stat_inner_ipld_obj &&
     test_cmp exp_stat_inner_ipld_obj actual_stat_inner_ipld_obj &&
     ipfs dag stat $HASH > actual_stat_ipld_obj &&
-    echo "Size: 61, NumBlocks: 2" > exp_stat_ipld_obj &&
+    echo "Size: 54, NumBlocks: 2" > exp_stat_ipld_obj &&
```

Comment:
iirc these differences come entirely from `{"data":123}` being encoded properly now, `a16464617461187b`:

```
a1                                                # map(1)
  64                                              #   string(4)
    64617461                                      #     "data"
  18 7b                                           #   uint(123)
```

whereas the current impl is doing the float thing again: `a16464617461fb405ec00000000000`:

a1                                                # map(1)
  64                                              #   string(4)
    64617461                                      #     "data"
  fb 405ec00000000000                             #   float(123)
```

so we're losing 7 bytes on that block, then the same number on the cumulative count for the block that links to it